src/main/java/com/hackclub/hccore/advancements/DiamondsAdv.java

```diff
@@ -19,13 +22,28 @@ public class DiamondsAdv extends BaseAdvancement {
             .description("Find your first diamond while mining");
     static int maxProgression = 1;
 
+    HCCorePlugin plugin;
+
     public DiamondsAdv(HCCorePlugin plugin, Advancement root, AdvancementKey key, CoordAdapter adapter) {
         super(key.getKey(), displayBuilder.coords(adapter, key).build(), root, maxProgression);
+        this.plugin = plugin;
 
         registerEvent(BlockBreakEvent.class, e -> {
            if (e.getBlock().getType() == Material.DIAMOND_ORE || e.getBlock().getType() == Material.DEEPSLATE_DIAMOND_ORE) {
                incrementProgression(e.getPlayer());
            }
         });
     }
+
+    @Override
+    public void giveReward(@NotNull Player player) {
+        player.sendMessage(ChatColor.GREEN
+                + "Congrats, you’ve found your very first diamond! You are now eligible for the exclusive (and limited edition!) Hack Club Minecraft stickers. Head over to "
+                + ChatColor.UNDERLINE + this.plugin.getConfig().getString("claim-stickers-url")
+                + ChatColor.RESET + ChatColor.GREEN + " to claim them!*");
+        player.sendMessage(ChatColor.GRAY + ChatColor.ITALIC.toString()
+                + "*This offer only applies to players who have never received the stickers before. If you have, please do not fill out the form again!");
+        player.sendMessage(ChatColor.GRAY + ChatColor.ITALIC.toString()
+                + "You will only see this message once.");
+    }
```

Comment:
for new msgs / code can the component api be used instead?
this format is deprecated